x-pack/plugins/observability_solution/investigate_app/common/schema/investigation_note.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import * as t from 'io-ts';
+
+const investigationNoteResponseSchema = t.type({
+  id: t.string,
+  content: t.string,
```

Comment:
notes can also have screenshot and links in future, maybe we need different schema for the `content` based on type: text | image | link, wdyt?

Reply:
Definitely, we will have to update the schema as we add support for more things. But I'm willing to delay that decision until the very day we work on this